webapp/components/PageFooter/PageFooter.vue

```diff
@@ -50,4 +50,9 @@ export default {
   margin-left: 0.2rem;
   margin-right: 0.2rem;
 }
+@media only screen and (max-width: 730px) {
+  .ds-footer {
+    display: none;
+  }
+}
```

Comment:
```suggestion
@media only screen and (max-width: 810px) {
  .ds-footer {
    display: none;
  }
}
```

Then the footer menu doeas not appear at the same time as the footer.